tests/cases/Integration/SendSms.phpt

```diff
@@ -0,0 +1,78 @@
+<?php declare(strict_types = 1);
+
+use Contributte\Gosms\Auth\AccessTokenClient;
+use Contributte\Gosms\Client\MessageClient;
+use Contributte\Gosms\Config;
+use Contributte\Gosms\Entity\Message;
+use Contributte\Gosms\Http\IHttpClient;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+use Nette\Neon\Neon;
+use Nette\Utils\Validators;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Tester\Assert;
+use Tester\Environment;
+
+require_once __DIR__ . '/../../bootstrap.php';
+
+$configPath = __DIR__ . '/../../gosms.neon';
+if (!is_file($configPath)) {
+	Environment::skip('No configuration file.');
+}
+
+$configData = Neon::decodeFile($configPath);
+
+/**
+ * @param array<string> $configData
+ * @see gosms.neon.example
+ */
+function createMessageClient(array $configData): MessageClient
```

Comment:
Neslo by to udelat pres ContainerLoader s vyuzitim extension? Otestovalo by se to tak cele.

Reply:
šlo... budu muset přidat do `dev` závislost na `nette/bootstrap`